.golangci.yaml

```diff
@@ -124,6 +130,11 @@ issues:
         - gocritic
       text: "unnecessaryDefer:"
 
+    - path: pkg/v1/tkr/controllers/source/suite_test.go
```

Comment:
Ditto.

Reply:
I got rid of this in rebase (I should've done a separate commit, made a mistake and rebased).